### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by tomit4** [aristocratos/btop#341] [General]
I love this application. It essentially has replaced conky on my Linux desktop. I'm excited to eventually use it to replace gpu monitoring tools like nvtop. I'd also like to put in a feature request for there to be a showing of wireless reception percentage (or whatever that's called).

Thanks for the wonderful application.

**Comment by Erotemic** [comment]
Is there any reason that btop wouldn't want to include this feature? I would like it too. It's annoying to have to run `btop` and `nvtop`. I'd rather that btop just show everything.
